### PR TITLE
feat: done with result page

### PR DESCRIPTION
**Pull request by Developers-codz**
Done with result page 
made pages responsive
direct link - https://deploy-preview-4--b-quiz.netlify.app/result

Thank you  for reviewing my PR :)

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
Nice and cleaner UI. Result page is great because of nicely of used gif . It was great. Codes are cleaner and readable. Nicely used classnames. Good job.

File: style.css [hunk 1]
```diff
@@ -156,4 +156,12 @@ footer {
   .card-category {
     margin: 1rem 0rem;
   }
```
Comment (kushanksriraj) [inline style.css:158]: margin: 1rem 0; ?

File: styles/result.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.result-wrapper {
+  min-height: 55vh;
+  text-align: center;
+  margin: 2rem;
+}
+.heading {
+  color: var(--primary);
```
Comment (kushanksriraj) [inline styles/result.css:7]: Lets add the name to the variable, like `--color-primary`

**Review by kushanksriraj — CHANGES_REQUESTED** [review]

**Review by VLeads — APPROVED** [review]
You have done an awesome job. Overall code looks great. It's simplicity level and design ideas are great !!

File: styles/result.css (hunk 2 above)
Comment (Developers-codz) [reply styles/result.css:7]: I already defined `--primary-color ` in my CL that's why just used `--primary` .

File: style.css (hunk 1 above)
Comment (Developers-codz) [reply style.css:158]: Done!!

File: styles/result.css (hunk 2 above)
Comment (kushanksriraj) [reply styles/result.css:7]: You can name it --color-default. Basically, the point is to know the what the variable is just by it's name, --primary doens't tell anything.

**Review by kushanksriraj — APPROVED** [review]

**Pull request merged**